plugins/cost-insights/src/components/AlertInsights/AlertStatusSummary.tsx

```diff
@@ -0,0 +1,116 @@
+import { default as SnoozeIcon } from '@material-ui/icons/AccessTime';
+import { ActionItemCard } from '../ActionItems';
+import { Alert, AlertStatus } from '../../types';
+import { useActionItemCardStyles as useStyles } from '../../utils/styles';
+
+type AlertStatusSummaryProps = {
+  open: boolean;
+  snoozed: Alert[];
+  accepted: Alert[];
+  dismissed: Alert[];
+};
+
+export const AlertStatusSummary = ({
+  open,
+  snoozed,
+  accepted,
+  dismissed,
+}: AlertStatusSummaryProps) => {
+  const classes = useStyles();
+
+  const isSnoozedListDisplayed = !!snoozed.length;
+  const isAcceptedListDisplayed = !!accepted.length;
+  const isDismissedListDisplayed = !!dismissed.length;
+
+  return (
+    <Collapse in={open}>
+      {isAcceptedListDisplayed && (
+        <Box p={1}>
+          {accepted.map((alert, index) => (
+            <Fragment key={`alert-accepted-${index}`}>
+              <ActionItemCard
+                disableScroll
+                alert={alert}
+                avatar={
+                  <Avatar className={classes.avatar}>
+                    {/* Icons indicate alert status. Do not hide from accesibility tree */}
+                    <AcceptIcon
+                      aria-hidden={false}
+                      role="img"
+                      aria-label={AlertStatus.Accepted}
+                    />
+                  </Avatar>
+                }
+              />
+              {index < accepted.length - 1 && <Divider />}
+            </Fragment>
+          ))}
+        </Box>
+      )}
+      {isSnoozedListDisplayed && (
+        <Box p={1}>
+          {snoozed.map((alert, index) => (
+            <Fragment key={`alert-accepted-${index}`}>
+              <ActionItemCard
+                disableScroll
+                alert={alert}
+                avatar={
+                  <Avatar className={classes.avatar}>
+                    <SnoozeIcon
+                      aria-hidden={false}
+                      role="img"
+                      aria-label={AlertStatus.Snoozed}
+                    />
+                  </Avatar>
+                }
+              />
+              {index < snoozed.length - 1 && <Divider />}
+            </Fragment>
+          ))}
+        </Box>
+      )}
+      {isDismissedListDisplayed && (
+        <Box p={1}>
+          {dismissed.map((alert, index) => (
+            <Fragment key={`alert-dismissed-${index}`}>
+              <ActionItemCard
+                disableScroll
+                alert={alert}
+                avatar={
+                  <Avatar className={classes.avatar}>
+                    <DismissIcon
+                      aria-hidden={false}
+                      role="img"
+                      aria-label={AlertStatus.Dismissed}
+                    />
+                  </Avatar>
+                }
+              />
+              {index < dismissed.length - 1 && <Divider />}
+            </Fragment>
+          ))}
+        </Box>
```

Comment:
A bit duplicated here, could be pulled into a component in the same file that accepts `<Alert[], Icon, AlertStatus, name>`?